packages/react-widgets/src/Multiselect.js

```diff
@@ -643,7 +648,7 @@ class Multiselect extends React.Component {
   }
 
   open() {
-    if (!this.props.open)
+    if (!this.props.open && (this.props.searchTerm.length >= this.props.minSearch))
```

Comment:
im a little worried about this, we want the widget to still open if the user clicks the open button i think

Reply:
but that kind of defies the point of the min search right? how would min search work if it opens when you click, no matter how long the search term?

Reply:
what if you want to just select something from the list?

Reply:
the default is `1` :P

Reply:
no default is 0

Reply:
sneaky

Reply:
so what do you think? :)

Reply:
idk i think if you never want this to open when the search is below a x we probably don't need this, since you can control searchTerm and always not allow the input to close when its <= whatever. The case where you don't want it to open while searching i think requires the change here, since you can't easily distinguish why `onToggle` is called. But I think your usecase doesn't case why it's called